vllm/engine/metrics.py

```diff
@@ -160,25 +166,16 @@ def __init__(self, labelnames: List[str], max_model_len: int):
             name="vllm:avg_prompt_throughput_toks_per_s",
             documentation="Average prefill throughput in tokens/s.",
             labelnames=labelnames,
+            multiprocess_mode="sum",
         )
         # Deprecated in favor of vllm:generation_tokens_total
         self.gauge_avg_generation_throughput = self._gauge_cls(
             name="vllm:avg_generation_throughput_toks_per_s",
             documentation="Average generation throughput in tokens/s.",
             labelnames=labelnames,
+            multiprocess_mode="sum",
         )
 
-    def _create_info_cache_config(self) -> None:
-        # Config Information
-        self.info_cache_config = prometheus_client.Info(
```

Comment:
Per @thekuffs's [comment](https://github.com/vllm-project/vllm/pull/7279/files#r1710464260), I think we could actually keep the info metrics, just use a gauge explicitly here?